packages/office-ui-fabric-react/src/components/Coachmark/Beak/Beak.tsx

```diff
@@ -0,0 +1,66 @@
+import * as React from 'react';
+import {
+  BaseComponent,
+  css,
+  classNamesFunction
+} from '../../../Utilities';
+import { IBeakProps } from './Beak.types';
+import { getStyles, IBeakStyles, IBeakStylesProps } from './Beak.styles';
+import { getComboBoxOptionClassNames } from 'src/components/ComboBox/ComboBox.classNames';
+
+export interface IBeakState {
+  left: string | null;
+  top: string | null;
+}
+
+export class Beak extends BaseComponent<IBeakProps, IBeakState> {
+  private _beakElement: HTMLElement;
+
+  constructor(props: IBeakProps) {
+    super(props);
+  }
+
+  public componentDidMount(): void {
+    const beakComputedStyles = window.getComputedStyle(this._beakElement);
+    this.setState({
+      left: beakComputedStyles.left,
+      top: beakComputedStyles.top
```

Comment:
Why do you compute these? I don't see it used anywhere

Reply:
Good catch, yeah they are not being used at all. It was an artifact from something else i was trying to work out.